config/locales/es.yml

```diff
@@ -91,6 +91,7 @@ es:
         no_results_message: No existen usuarios con el nombre empezando por %{query}
         results_message: Usuarios cuyo nombre empieza por %{query}
     nav:
+      active_topics:
```

Comment:
Please remove the empty translations and then run:

```bash
GOOGLE_TRANSLATE_API_KEY=<YOUR_API_KEY> i18n-tasks translate-missing
```

You can obtain an API key at the [Google API Console](https://code.google.com/apis/console) under Project -> APIS & AUTH -> Credentials -> Public API access -> Key for server applications.